tests/local-overlay-store/common.sh

```diff
@@ -0,0 +1,83 @@
+source ../common.sh
+
+requireEnvironment () {
+  requireSandboxSupport
+  [[ $busybox =~ busybox ]] || skipTest "no busybox"
+  if [[ $(uname) != Linux ]]; then skipTest "Need Linux for overlayfs"; fi
```

Comment:
This should probably test whether the overlayfs kernel module is loaded or overlayfs is in `/proc/filesystems`.

Reply:
Weird, it didn't start appearing or me on the list on one of my machines until I made one.

Reply:
Maybe because the kernel module was lazily loaded?

Reply:
That would make sense. I guess we can try loading it?